neo/renderer/tr_main.cpp

```diff
@@ -599,16 +585,16 @@ bool R_RadiusCullLocalBox( const idBounds &bounds, const float modelMatrix[16],
 
 	R_LocalPointToGlobal( modelMatrix, localOrigin, worldOrigin );
 
-	worldRadius = (bounds[0] - localOrigin).Length();	// FIXME: won't be correct for scaled objects
+	worldRadius = ( bounds[0] - localOrigin ).Length();	// FIXME: won't be correct for scaled objects
 
 	for ( i = 0 ; i < numPlanes ; i++ ) {
 		frust = planes + i;
 		d = frust->Distance( worldOrigin );
+
```

Comment:
please don't commit all these unrelated whitespace changes. use `git gui` or similar to only commit the relevant changes

Reply:
sorry been hammering away at getting it to compile on gcc / clang.
mingw-w64 builds work now as well as windows clang builds.

there was one sorry bugger with current mingw-w64-abi the __cpuid function now uses the same syntax as msvc's so i had to change the #if defined(__GNUC__) && (defined(__i386__) || defined (__x86_64__)) guard into one specific for unix  so the mingw builds dont use that one.

well atleast it was informal i reckon id make a new repo after getting the last build bugs out and start slowly adding in the changes, that way we can avoid all the whitespace changes from me mucking around in the code.

Reply:
for now lets use it as a draft to iron out build bugs on different arch's, bit easier than having multiple operating systems 
:) when it is all up and running ill retire the fork and start from a fresh one adding only the changes nessesary without the formatting. also gives you the option to chime in before we make changes that may make or break something.

Reply:
ok seems to build on ubuntu now atleast in the one in wsl.
running it on wsl does not seem to work though the game window opens it crashes shortly after with a crash in imgui (not sure wsl is the best testing environment).

i packed up the binaries here https://sourceforge.net/projects/cbadvanced/files/Game%20Projects/dhewm3-ubuntu.7z/download

could you try them out on a real linux box maybe ?.

Reply:
https://sourceforge.net/projects/cbadvanced/files/Game%20Projects/dhewm3-git.tar.zst/download

my current source code ignore whitespace it is not ment to be merged as is only for testing if it works :).

https://sourceforge.net/projects/cbadvanced/files/Game%20Projects/dhewm3.tar.zst/download

i forgot linux sets some flags that are probably lost when zipping it with a windows tool so heres a tarball of the game executables made in ubuntu, i used the linux dist dir in the source as install target (not sure if correct ?).